front/app/components/ProjectAndFolderCards/index.tsx

```diff
@@ -15,49 +16,64 @@ import { PublicationStatus } from 'services/projects';
 
 export type PublicationTab = PublicationStatus | 'all';
 
-export default ({ publicationStatusFilter, ...otherProps }: BaseProps) => {
+export interface Props {
+  showTitle: boolean;
+  layout: TLayout;
+  allowedPublicationStatuses: BaseProps['publicationStatusFilters'];
+}
+
+export default ({ allowedPublicationStatuses, ...props }: Props) => {
```

Comment:
Also to still improve imo: use named components instead of anonymous exports. This can make debugging easier.